spec/config/initializers/ab_tests_spec.rb

```diff
@@ -4,11 +4,9 @@
   describe '#all' do
     it 'returns all registered A/B tests' do
       expect(AbTests.all).to match(
-        {
-          ACUANT_SDK: an_instance_of(AbTest),
-          DOC_AUTH_VENDOR: an_instance_of(AbTest),
-
-        },
+        ACUANT_SDK: an_instance_of(AbTest),
+        DOC_AUTH_VENDOR: an_instance_of(AbTest),
+        RECAPTCHA_SIGN_IN: an_instance_of(AbTest),
```

Comment:
we made some adjustments in this other PR https://github.com/18F/identity-idp/pull/11104#discussion_r1725942358

same feedback here

Reply:
Makes sense to me, updated in eea9e080d5